metals/src/main/scala/scala/meta/internal/metals/debug/ExtendedScalaMainClass.scala

```diff
@@ -61,10 +70,68 @@ object ExtendedScalaMainClass {
     s"$safeJavaHome $jvmOptsString -classpath \"$classpathString\" $mainClass $argumentsString"
   }
 
+  /**
+   * Create a manifest jar containing the classpath.
+   * It's created in `.metals/tmp` woth name derived
+   * from the classpath MD5 sum so that it's not created
+   * every time.
+   *
+   * @param classpath classpath to put into the jar file
+   * @param workspace current workspace path
+   * @return path to the create jar as string
+   */
+  private def tempManifestJar(
+      classpath: List[String],
+      workspace: AbsolutePath,
+  ): String = {
+
+    val classpathDigest = MD5.compute(classpath.mkString)
+    val manifestJar =
+      workspace
+        .resolve(Directories.tmp)
+        .resolve(s"classpath_${classpathDigest}.jar")
+
+    if (!manifestJar.exists) {
+
+      manifestJar.touch()
+      manifestJar.toNIO.toFile().deleteOnExit()
+
+      // Add trailing slash to directories so that manifest dir entries work
```

Comment:
Maybe I'm totally missing it, but where does this happen? Looking in `encode` I don't see that happening.

Reply:
Because I removed it and just settled on URI :sweat:

Reply:
Removed!